docs/Building.md

```diff
@@ -101,8 +101,8 @@ container image.  The tests cover multiple topologies and covers most of the ava
 options.
 
 The full suite of system tests takes approximately 30 minutes to complete.  If you are developing code for Calico
-Docker, and are regularly running the system tests, a faster subset of system tests may be run which covers most
-mainline function.
+Docker, and are regularly running the system tests, you can run a subset of the tests instead.
+(See both examples below.)
```

Comment:
Rather than having that appended bit, why not just link <run a subset of the tests> to the heading below.